packages/react-components/react-dialog/src/stories/DialogNoFocusableElement.stories.tsx

```diff
@@ -0,0 +1,42 @@
+import * as React from 'react';
+import { Dialog, DialogTrigger, DialogSurface, DialogTitle, DialogBody } from '@fluentui/react-dialog';
+import { Button } from '@fluentui/react-components';
+
+export const NoFocusableElement = () => {
```

Comment:
tbh I wouldn't even want to put this example in the docsite

Reply:
The same way as `Nested Dialogs`, although it's not ideal, it's good to document edge cases.